### DISCUSSION TITLE
Can't follow user of my new instance

### DISCUSSION THREAD

**Discussion by aeharding** [mastodon/mastodon#22779] [Q&A] [answered]
Hello!

I've created a new instance, https://social.harding.dev.

I can access it fine and post new toots, and lookup users on other instances but I am having trouble doing anything else. Whenever I try to follow an external user it gets stuck in "pending follow request".

Additionally, whenever I try to follow the user on my instance (@alex@harding.dev) **from a different instance,** I get "503 Remote data could not be fetched".

Account name: @alex@harding.dev 
Server installation URL: https://social.harding.dev

I have tried to properly set up `.well-known` and have configured `LOCAL_DOMAIN` and `WEB_DOMAIN`. At this point I'm stuck. Any help appreciated!

**Comment by rbairwell** [comment]
Hi @aeharding ,

Are you still having this problem? I've just done a test from my testbed and everything seemed to go okay - and a follow from my @rbairwell@mastodon.org.uk account seemed to work and I was able to follow you.

**Reply by aeharding** [reply]
I'm still having this problem unfortunately, and I don't see any evidence that you followed me :(

Here's more errors that I see from sidekiq:

<img width="472" alt="image" src="https://user-images.githubusercontent.com/2166114/209684355-06e328c2-26e5-40dd-85d0-56b90c601fd7.png">

Also, my home feed is stuck like this:

<img width="486" alt="image" src="https://user-images.githubusercontent.com/2166114/209684429-213be895-73f5-4c0d-994e-f426661e6890.png">

And when I go to another server to follow myself, I still get a 503 when I try to look up @alex@harding.dev:

<img width="498" alt="image" src="https://user-images.githubusercontent.com/2166114/209684544-4cca3af9-f227-46cf-b089-ac5b0474e9b3.png">

**Reply by aeharding** [reply]
Well it looks like I spoke too soon, because the follow just popped up in my notifications:

<img width="661" alt="image" src="https://user-images.githubusercontent.com/2166114/209684932-f0773712-f8f4-49b6-b151-abeb193f0e52.png">

Unfortunately, the other problems remain, and I'm unable to search for my user from urbanists.social or climatejustice.rocks

**Comment by rbairwell** [comment, marked as answer]
it's interesting that I've (just) tried to follow you from 5 different servers and 2 of them worked... Have you got a firewall running on the server (you should have) and has it block any instances?

Are you running any kind of WAF (such as mod security), filtering or proxy/cache server in front of it?

I notice it is running on a Spectrum Residential IP address: do they allow/permit you to run servers?

**Reply by aeharding** [reply]
Oh wow, that was it! I have a pretty strict country firewall by default that I had to allow mastodon. Thanks!!

**Comment by rbairwell** [comment]
Just noticed that the server only supports TLS v1.3 : can you try tweaking that in nginx to alliw TLS v1.2 as well to see if it makes a difference?

**Reply by aeharding** [reply]
Just documenting if anyone is looking up this issue: I still have TLS v1.3 only enabled and everything seems to be fine now (the problem was the country geoip block)

**Reply by redegelde** [reply]
got same error, you think its the geoip from the server. Can you point me to the direction how to fix this.
or do i have to ask the server hoster

**Reply by rbairwell** [reply]
> got same error, you think its the geoip from the server. Can you point me to the direction how to fix this. or do i have to ask the server hoster

It's reasonably unusual for hosts to have Geoip blocking in place (and then they only tend to block countries outside their demographics or where high levels of abuse/spam-like activity come from - such as Russia, China and India).

Which instance/server are you on? I assume you can view your home feed on the server without problems and it is just following other people which is the issue (if you can't reach the server, we'll need to know which country you are in/from).

**Reply by redegelde** [reply]
my own server is https://mastodon.education user redegelde

and from this server i can not find this user
@wenslauw@m.beautifullosers.org

**Reply by rbairwell** [reply]
If you are searching for @wenslauw@m.beautifullosers.org from your server (it does appear m.beautifullosers.org was down for a short while today/yesterday) and nothing is appearing in the results, just try https://m.beautifullosers.org/@wenslauw and see if that makes a difference (in both situations give it a few seconds to allow to query).

I assume that you are getting no results from that search and that you cannot access https://m.beautifullosers.org/about from your server using curl/wget etc.

Your instance - mastodon.education - appears to be hosted via Time4VPS [ AS62282 and AS212531 ] in Lithuania with m.beautifullosers.org on 195.201.119.111 in Hetzner in Germany.

Doing a ping test from your providers Looking Glass at https://lg.time4vps.com/ (good on them for making it available) shows 100% packet loss on a ping tom.beautifullosers.org and the mtr shows 100% failures either side of 12064.your-cloud.host/116.203.161.20 (from my UK address, I get a failure before that point, but the continues onto the host). If you run mtr/traceroute/ping from your server would should see the packets fail before they reach 195.201.119.111.

Checking other Lithuanian based looking glasses such as https://lg.ntt.lt/ [ AS33922 ] , https://lg.bite.lt/ [ AS13194 ] all show failures. http://lg.evio.pl/ [ AS57665 ] in Poland also shows failures. https://lg.telstraglobal.com/ [ AS4637] from Australia and India also shows failure

Using other Looking Glasses such as https://lg.leasewebstatus.com/ from Amsterdam, France and US all show success. https://lg.ovh.net/traceroute/bhs+rbx+sbg+gra/ipv4?q=195.201.119.111 shows success from France and Canada.

Based on this data, it looks like the server m.beautifullosers.org is purposely blocking a number of IP ranges (so not targeted at your provider or your country and the issue probably isn't network related as all tests are reaching the host before the destination). All I can 

All I can suggest at this point is to contact the admin of https://m.beautifullosers.org/about (who you are actually trying to follow) either via another server/instance (incidents like this is why I have 7 accounts on different servers!) or via email (their email address is shown on the https://m.beautifullosers.org/about page which can be accessed indirectly via services such as https://archive.is . It might be worth pointing them towards this post as it might save them quite a bit of diagnostics time ;)

**Comment by alisyd** [comment]
I am facing the same issue, I migrated my mastodon instance to a new server, but I cannot follow people from other instances on my new server but I can do so on my old server.

**Comment by Usman-52** [comment]
I'm not able to follow other people from my account, idk why? kindly someone help me
